### PR TITLE
Add Morpho, DSR & Curve

### PR DESCRIPTION
**Pull request by akshatmittal**

### CODE REVIEW

File: src/utils/plugins/mainnet.ts [hunk 1]
```diff
@@ -433,42 +439,42 @@ const plugins: CollateralPlugin[] = [
...
-    decimals: 18,
-    targetUnit: TARGET_UNITS.USD,
-    referenceUnit: 'DAI',
-    collateralToken: 'DAI',
-    description: '',
-    depositContract: '0xe8461dB45A7430AA7aB40346E68821284980FdFD',
-    collateralAddress: underlyingCollateralAddresses.DAI,
-    rewardToken: [CVX_ADDRESS[ChainId.Mainnet], CRV_ADDRESS[ChainId.Mainnet]],
-  },
+  // {
```
Comment (lcamargof) [inline src/utils/plugins/mainnet.ts:442]: Why are these ones commented?

Comment (akshatmittal) [reply src/utils/plugins/mainnet.ts:442]: I thought they weren't deployed yet, but looking back at the deployment output I see them being deployed as well. Will update.

Comment (akshatmittal) [reply src/utils/plugins/mainnet.ts]: Updated it, but I'd like at least one other person to verify the addresses.

**Review by lcamargof — APPROVED** [review]
lgtm

**Pull request merged**